### PR TITLE
Group Josh and Tracy Solution

### PR DESCRIPTION
**Pull request by JayBk**
I was having trouble accessing my account before 9am. Sorry.

### CODE REVIEW

**Review by wasjosh — COMMENTED** [review]
Nice work!

File: Makefile [hunk 1]
```diff
@@ -7,6 +7,14 @@ test:
 	@echo $(TAG)Running tests$(END)
 	PYTHONPATH=. py.test tests
 
+test-operations:
```
Comment (vkotek) [inline Makefile:10]: Didn't think of this, will use this next time! Thanks

File: calculator/operations.py [hunk 2]
```diff
@@ -1,23 +1,19 @@
...
 
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    return functools.reduce(lambda x, y: x-y, args)
```
Comment (vkotek) [inline calculator/operations.py:8]: Your lambda functions are so simple and beautiful. Gotta get more comfortable with those myself!

Comment (JayBk) [reply calculator/operations.py:8]: Tracy came up with this idea!

**Pull request closed without merging**